### PR TITLE
chore: CI workflow bugfixes

### PR DESCRIPTION
**Pull request by joeldodge79**

### CODE REVIEW

File: .github/workflows/codegen-ci.yml [hunk 1]
```diff
@@ -98,7 +98,6 @@ jobs:
           path: results/sdk-codegen
 
   publish-test-results:
-    name: Codegen Test Results
```
Comment (joeldodge79) [inline .github/workflows/codegen-ci.yml:101]: confusing and not needed

File: .github/workflows/codegen-ci.yml [hunk 2]
```diff
@@ -126,15 +125,15 @@ jobs:
   # there will be duplicate PR check names but github handles this and requires
   # both to be passing.
   noop-python-results:
-    name: Python Test Results
+    name: Python Tests
```
Comment (joeldodge79) [inline .github/workflows/codegen-ci.yml:128]: oops, these need to match the `check_name` property of each action

File: .github/workflows/tssdk-ci.yml [hunk 3]
```diff
@@ -173,7 +172,7 @@ jobs:
       - name: Publish Unit Test Results
         uses: EnricoMi/publish-unit-test-result-action@v1.12
         with:
-          check_name: Test Results
+          check_name: Typescript Tests
```
Comment (joeldodge79) [inline .github/workflows/tssdk-ci.yml:175]: missed this one last time

**Review by jkaster — APPROVED** [review]

**Review by joeldodge79 — COMMENTED** [review]
as always, much tweaking involved

**Review by jkaster — APPROVED** [review]

**Pull request merged**